README.md

```diff
@@ -17,3 +71,21 @@ const { HTTPClient, registerMetrics } = require("@loke/http-client");
 const promClient = require("prom-client");
 registerMetrics(promClient.register);
 ```
+
+## Breaking Changes in v2
+
+The main breaking change is the we switched from `got` to native node `fetch`
+for the underlying HTTP client. This means that the API is now more aligned with
+the native fetch API.
+
+- The Response object is now from native `fetch` and not `got`
+- Options are now native `fetch` options not `got` options
+- Errors are now native `fetch` errors not `got` errors
```

Comment:
Are there some common error cases we should document?

Reply:
Looking the deliverect PR I think a big one would be `statusCode` -> `status`

Reply:
and no more `.body` or `.headers`, use `.response` instead

Reply:
Mostly because all the error conditions are harder to hit when testing

Reply:
I think the example `_handlerError` does a pretty good job showing what the basic pattern should be